### ISSUE TITLE
IloAccounts -j argument not working in 3.1.0

### ISSUE THREAD

**Issue by john159753** [HewlettPackard/python-redfish-utility#31]
After upgrading to 3.1.0, the -j and --json argument in iloaccounts doesnt work.

```
C:\Program Files\Hewlett Packard Enterprise\RESTful Interface Tool>ilorest iloaccounts -j
iLOrest : RESTful Interface Tool version 3.1
Copyright (c) 2014-2020 Hewlett Packard Enterprise Development LP
--------------------------------------------------------------------------------
usage: iLOrest iloaccounts [-h] [--url URL] [-u USER] [-p PASSWORD]
                           [--https HTTPS_CERT] [--includelogs] [--path PATH]
                           [--logout] [-j]
                           {default,add,modify,changepass,delete,addcert,deletecert}
                           ...
iLOrest iloaccounts: error: The option -j is not available for iloaccounts
```

**Comment by rajeevkallur** [comment]
This will be fixed in 3.1.1 soon to be released.

**Comment by rajeevkallur** [comment]
This is fixed. Please use latest release.
